src/Microsoft.TestPlatform.Extensions.BlameDataCollector/CrashDumperFactory.cs

```diff
@@ -37,6 +37,20 @@ public ICrashDumper Create(string targetFramework)
                     return new ProcDumpCrashDumper();
                 }
 
+                // On net5.0 we don't have the capability to crash dump on exit, which is useful in rare cases
+                // like when user exits the testhost process with a random exit code, adding this evn variable
+                // to force using procdump. This relies on netclient dumper actualy doing all it's work in blame collector
+                // where it sets all the environment variables, so in effect we will have two crash dumpers active at the same time.
+                // This proven to be working okay while net5.0 could not create dumps from Task.Run, and I was using this same technique
+                // to get dump of testhost. This needs PROCDUMP_PATH set to directory with procdump.exe, or having it in path.
+                var procdumpOverride = Environment.GetEnvironmentVariable("VSTEST_DUMP_FORCEPROCDUMP")?.Trim();
+                var forceUsingProcdump = string.IsNullOrWhiteSpace(procdumpOverride) && procdumpOverride != "0";
+                if (forceUsingProcdump)
+                {
+                    EqtTrace.Info($"CrashDumperFactory: This is Windows on {targetFramework}. Forcing the use of ProcDumpCrashDumper that uses ProcDump utility, via VSTEST_DUMP_FORCEPROCDUMP={procdumpOverride}.");
+                    return new ProcDumpCrashDumper();
```

Comment:
you are not covering it with tests?

Reply:
I will at some point, almost nothing from blame has integration tests in this repo. But I have a separate pipeline that I use for validation, it takes a lot of time to run though.